scripting/get5.sp

```diff
@@ -112,6 +112,10 @@ char g_FavoredTeamText[MAX_CVAR_LENGTH];
 int g_PlayersPerTeam = 5;
 int g_MinPlayersToReady = 1;
 int g_MinSpectatorsToReady = 0;
+int g_RoundNumber = 0;
```

Comment:
This value is likely to be incorrect with backup/restores. I would avoid storing it at all, and derive it otherwise (there are some netprops to calculate it, used somewhere else in the plugin).